cudamapper/src/index_gpu.cuh

```diff
@@ -563,6 +577,57 @@ IndexGPU<SketchElementImpl>::IndexGPU(std::shared_ptr<DeviceAllocator> allocator
                    filtering_parameter);
 }
 
+template <typename SketchElementImpl>
+IndexGPU<SketchElementImpl>::IndexGPU(std::shared_ptr<DeviceAllocator> allocator,
```

Comment:
Nitpicky: I'm not sure, but I think we generally put allocator as the last parameter

Reply:
actually I found everywhere it's passed as the first parameter, wrote it the same way to be consistent with the rest of the code.

Reply:
Oh, right, sorry, allocators are a new addition, still didn't get used to them